README.md

```diff
@@ -12,70 +12,193 @@ Airport Challenge
-
-As the system designer
-So that the software can be used for many different airports
-I would like a default airport capacity that can be overridden as appropriate
-
-As an air traffic controller
-To ensure safety
-I want to prevent landing when the airport is full
-
-As an air traffic controller
-So I can get passengers on the way to their destination
-I want to instruct the airport to let a plane take off and confirm that it is no longer in the airport
-
-As an air traffic controller
-To avoid confusion
-I want to prevent asking the airport to let planes take-off which are not at the airport, or land a plane that's already landed
-```
-
-#### Extended Acceptance Criteria
-```
-As an air traffic controller
-To ensure safety
-I want to prevent takeoff when weather is stormy
-
-As an air traffic controller
-To ensure safety
-I want to prevent landing when weather is stormy
-
-As an air traffic controller
-To count planes easily
-Planes that have landed must be at an airport
-```
-
-Your task is to test drive the creation of a set of classes/objects to satisfy all the above user stories. You will need to use a random number generator to set the weather (it is normally sunny but on rare occasions it may be stormy). In your tests, you'll need to stub random behaviour to ensure consistent test behaviour.
-
-Your code should defend against [edge cases](http://programmers.stackexchange.com/questions/125587/what-are-the-difference-between-an-edge-case-a-corner-case-a-base-case-and-a-b) such as inconsistent states of the system ensuring that planes can only take off from airports they are in; planes that are already flying cannot take off and/or be in an airport; planes that are landed cannot land again and must be in an airport, etc.
+## User Story 1  
+As an air traffic controller  
+So I can get passengers to a destination  
+I want to instruct the airport to land a plane  
+
+**Domain Model 1**
+
+| Objects | Properties                    | Messages          | Outputs |
+| ------- | ----------------------------- | ----------------- | ------- |
+| Airport |planesAtAirport @Array[@Plane] | planeLands(@Plane)| @Void   |
+|         |airportName@String             |                   |         |
+| Plane   |planeID @String                |                   |         |
+
+**Tests**  
+*Test 1* - Check that only Plane types are able to land at the Airport when planeLands  
+*Test 2* - Length of planesAtAirport array at the Airport increases by 1 when planeLands  
+*Test 3* - Check that edge cases such as 'null' as a type are unable to land at the airport and length of planesAtAirport remains at 0  
+*Test 4* - Check that when something that isn't a plane attempts to land at the airport, a message is returned  
+*Test 5* - When a new airport instance is created from the Airport class, it contains an airportName property  
+*Test 6* - When a new plane instance is created from the plane class, it contains a planeID property  
```

Comment:
Should I have a created another test that checks planeID property is undefined when an ID is not provided? Is there a  way to block the creation of an instance if a value is not given?